### PR TITLE
Ember 3.0 Release Blog Post

### PR DESCRIPTION
**Pull request by locks**
- [Rendered](https://github.com/emberjs/website/blob/3-0-release-blog-post/source/blog/2018-02-14-ember-3-0-released.md)
- Preview
- 3.1-beta blog post: https://github.com/emberjs/website/pull/3187

## Sign-offs

- [ ] Ember.js (@chancancode)
- [ ] Ember Data (@bmac)
- [ ] Ember CLI (@rwjblue)
- [ ] Website (@locks, @mixonic)

## Checklist

- Ember
  - [ ] Blog
  - [ ] v3.0.0
  - [ ] v3.1.0-beta.1
- Ember Data
  - [ ] Blog
  - [ ] v3.0.0
  - [ ] v3.1.0-beta.1
- Ember CLI
  - [ ] Blog
  - [ ] v3.0.0
  - [ ] v3.1.0-beta.1

See https://github.com/emberjs/website/pull/2824/files for inspiration.

### CODE REVIEW

File: source/blog/2017-02-12-ember-3-0-released.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+Each deprecation has an entry in the deprecation guide describing the migration
+path to more stable API. Deprecated public APIs are not removed until a major
+release of the framework.
+
+Consider using the
+[ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow)
+addon if you would like to upgrade your application without immediately addressing
+deprecations.
+
+Two new deprecations are introduces in Ember.js 3.0:
```
Comment (loadx) [inline source/blog/2017-02-12-ember-3-0-released.md]: s/introduces/introduced

**Comment by rwjblue** [comment]
Should mention https://github.com/rondale-sc/ecma5-getter-ember-codemod in the section about native ES5 getters.

**Comment by mixonic** [comment]
So this is a reasonable pass at a 3.0 release post. We're going to spin out 3.1 beta into a distinct post to try and keep people focused on the 2.x - 3.x transition.

I'd like to give an example of the steps for upgrading an app (really just normal upgrade steps).

File: source/blog/2018-02-12-ember-3-0-released.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+In Ember 3.0 we've changed the default blueprint for generated tests to use the
+new testing API specified in [RFC #232](https://github.com/emberjs/rfcs/blob/master/text/0232-simplify-qunit-testing-api.md)
+and [RFC #268](https://github.com/emberjs/rfcs/blob/master/text/0268-acceptance-testing-refactor.md).
+
+You can find the final documentation for these test helpers at:
+
+* https://github.com/emberjs/ember-test-helpers/blob/master/API.md
+
+And additionally the [Ember.js Testing Guides](TODO) have been updated.
```
Comment (mixonic) [inline source/blog/2018-02-12-ember-3-0-released.md]: TODO

File: source/blog/2018-02-12-ember-3-0-released.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+```bash
+> yarn global add ember-cli # Or npm install -g ember-cli
+> ember new my-project
+> cd my-project
+> ember serve # Then visit http://localhost:4200
+```
+
+Additionally, today the 3.1 beta cycle begins for these same projects. We've
+split out the beta release details into a separate blog post to keep this one
+focused on app migration steps and guidance.
```
Comment (mixonic) [inline source/blog/2018-02-12-ember-3-0-released.md]: Phrasing may need tweaking based on if the 3.1-beta post is pending, etc.

**Comment by mixonic** [comment]
Before this goes live we need guides and api docs to be deployed.

**Pull request merged**